composer.json

```diff
@@ -131,6 +131,7 @@
         "moontoast/math": "Required to use ordered UUIDs (^1.1).",
         "nyholm/psr7": "Required to use PSR-7 bridging features (^1.2).",
         "pda/pheanstalk": "Required to use the beanstalk queue driver (^4.0).",
+        "predis/predis": "Required to use the redis cache and queue drivers (^1.1.2).",
```

Comment:
We should say only to use the predis connector for those things. Use of the extension should be preferred.

Reply:
I don't think it hurts to restore it like it was before. Now that Predis is maintained again, people can choose what they want to do. We could note in the docs that we favor the PHP extension.

Reply:
Hmm actually, no you're right. I'll adjust this.